lib/ae_mdw/db/model.ex

```diff
@@ -180,22 +179,20 @@ defmodule AeMdw.Db.Model do
   @type id_count() :: record(:id_count, index: id_count_index(), count: non_neg_integer())
 
   # object origin :
-  #     index = {tx_type, pubkey, tx_index}, tx_id = tx_hash
-  @origin_defaults [index: {nil, nil, nil}, tx_id: nil]
```

Comment:
Wouldn't it be a better to leave the tx_hash field in the record? Unless the idea is for it to be only in the RevOrigin table to not duplicate the data

Reply:
It alread holds {txi, idx} tuple, and you can compute the tx_hash using the txi, no need to store redundant data, specially since hashes take a lot more space